nameifier.go

```diff
@@ -52,8 +45,8 @@ func NewNameifier(basePath string) *Nameifier {
 	n := &Nameifier{
 		BasePath: basePath,
 	}
-	loadJson(n.BasePath+"/"+NounsFilename, n)
-	loadJson(n.BasePath+"/"+AdjectivesFile, n)
+	loadJson(nouns, n)
+	loadJson(adjectives, n)
```

Comment:
if `BasePath` isn't used for anything else, can we remove it, too?

Reply:
yeah, you're right. I'll remove it.